swift/Sources/OpenSpiel/GridMaze.swift

```diff
@@ -396,43 +403,43 @@ public struct GridCell {
 public func SPACE(reward: Double = -1) -> GridCell {
   return GridCell(
     oneWordDescription: "SPACE", reward: reward,
-    isInitial: false, isTerminal: false,
+    isInitial: false, isTerminal: false, isGoal: false,
```

Comment:
These `GridCell` factories are more idiomatically written as static methods/properties:
```swift
extension GridCell {
  /// Creates a start cell with the given reward.
  static func start(reward: Double = -1) -> GridCell { fatalError() }
  /// Creates a goal cell with the given reward.
  static func goal(reward: Double) -> GridCell { fatalError() }
  /// Creates a hole cell with the given reward.
  static func hole(reward: Double) -> GridCell { fatalError() }
  /// Creates a wall cell with the given reward.
  static func wall(reward: Double) -> GridCell { fatalError() }
  /// A bounce cell.
  static var bounce: GridCell { fatalError() }
}
```

Reply:
SGTM. Changed code and doc in the upcoming commit.